tests/tutor/test_tutor_teacher.py

```diff
@@ -0,0 +1,1473 @@
+"""Test the Tutor teacher functions."""
+
+from tests.markers import nondestructive, test_case, tutor, expected_failure
```

Comment:
The markers should be in sorted order:
`from tests.markers import expected_failure, nondestructive, test_case, tutor`